docs/how-to/manage-roles.md

```diff
@@ -0,0 +1,50 @@
+Roles are groupings of [policies](/how-to/manage-policies) that are assigned to [users](/how-to/manage-users) to manage authorization.
+
+Listing roles
+----
+
+```
+$ fleet auth role list
+Name       Policies
+---------  ----------
+Admin      AllowAll
+JuniorDev  AllowAll, DenyProdEnv
+```
+
+Creating a role
+----
+
+You can optionally provide policies with a comma separated list to include in a role when you create it or you can add them later.
+
+```bash
+$ fleet auth role add Admin
+Added role: Admin
+$ fleet auth role add Admin --policies AllowAll
+Added role: Admin
+$ fleet auth role add JuniorDev --policies "AllowEnvironments, AllowReleases, DenyProdEnv"
+Added role: JuniorDev
+```
+
+Removing a role
+----
+
+```
+$ fleet auth role remove Admin
+Removed role: Admin
+```
+
```

Comment:
With the exception of auth certs, creation and deletion of certain objects, e.g. a user's roles or a role's policies, does not depend on the (non-)existence of their related object and visa versa. This could be done but at present such code would be very brittle.